tests/pyrog/factories.py

```diff
@@ -30,7 +30,6 @@ class Meta:
     id = factory.Sequence(lambda n: f"resource_id_{n:04d}")
     source = factory.SubFactory(SourceFactory)
     primary_key_owner = factory.SubFactory("tests.pyrog.factories.OwnerFactory")
-    logical_reference = factory.Sequence(lambda n: f"logical_reference_{n:04d}")
```

Comment:
I would prefer this (on my branch to fix django_db tests: https://github.com/arkhn/fhir-river/commit/49354cb39107c2b9fc156c4d087c35fa9306463a